### PR TITLE
New package: heroic-games-launcher-2.4.3

### PR DESCRIPTION
**Pull request by wiktorciurej**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64*
  - armv7l*
  - x86_64-musl

### CODE REVIEW

**Comment by Johnnynator** [comment]
https://github.com/void-linux/void-packages/actions/runs/3353335290/jobs/5558484193#step:7:170

You are fetching precompiled binaries for glibc.
Also heroic would be pretty limited on *musl and arm/aarch64 since everything it downloads is for x86_64-glibc.

File: srcpkgs/heroic-games-launcher/files/heroic.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Heroic Games Launcher
+Exec=/usr/bin/heroic %U
```
Comment (sirkhancision) [inline srcpkgs/heroic-games-launcher/files/heroic.desktop:3]:
```suggestion
Exec=/opt/heroic/heroic %U
```

File: srcpkgs/heroic-games-launcher/template [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Template file for 'heroic-games-launcher'
+pkgname=heroic-games-launcher
+version=2.4.3
```
Comment (sirkhancision) [inline srcpkgs/heroic-games-launcher/template:3]:
```suggestion
version=2.6.1
```

File: srcpkgs/heroic-games-launcher/template [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+revision=1
+archs="x86_64 armv7l aarch64 ~*-musl"
+wrksrc="HeroicGamesLauncher-${version}"
```
Comment (sirkhancision) [inline srcpkgs/heroic-games-launcher/template:6]:
```suggestion
archs="x86_64"
```

File: srcpkgs/heroic-games-launcher/template [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+hostmakedepends="nodejs python3 yarn"
+depends="alsa-lib gtk+3 nss python3"
```
Comment (sirkhancision) [inline srcpkgs/heroic-games-launcher/template:8]:
```suggestion
depends="fuse gawk"
```

File: srcpkgs/heroic-games-launcher/template [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+checkdepends="tar xz"
+short_desc="Open Source Launcher for GOG and Epic Games"
+maintainer="Wiktor Ciurej <wiktor.ciurej@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher"
+distfiles="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/archive/v${version}.tar.gz"
+checksum=b6590fd99776c2f6d890266ee177d1d90f8a127eaa47b9aec41fc00c4194bc9a
```
Comment (sirkhancision) [inline srcpkgs/heroic-games-launcher/template:15]:
```suggestion
checksum=d312ae93a63029491bf23eb52074580e4a1830d030b70f9547fd9b187306f8aa
```

**Comment by sirkhancision** [comment]
The above suggestions are to update it to the latest version, reduce dependencies, simplify the process and make it build properly (the base code wasn't building in my machine). Compilation also failed when building for anything other than x86_64, so I've removed the other architectures.

**Pull request closed without merging**